.circleci/src/workflows/scheduled_native_development_delivery.yml

```diff
@@ -41,30 +52,71 @@ jobs:
         - tuerantuer-android
       production_delivery: false
       requires:
+        - check_native
         - e2e_android
+        - build_integreat_android
+
+#  - build_android:
```

Comment:
should there no malte development delivery in general or did you just comment this part out for now?

Reply:
Just for now. RIght now, we're still in the setup of the stores (or at least I am for ios). Once that is done, this has to be uncommented. There is an issue for this though: https://issues.tuerantuer.org/browse/IGAPP-253

E.g. the webapp delivery is done for malte as well.